src/IProfile.jl

```diff
@@ -96,6 +96,7 @@ function insert_profile_block(fblock::Expr, tlast, tnow, timers, blast, bnow, by
     end
     descend = PROFILE_STATE & PROFILE_DESCEND > 0
     fblocknewargs = []
+    local lasttag
```

Comment:
Was curious how this ever worked, did scoping rules for for loops change? I would've thought `lasttag` would've been undefined on line 113 (now 114), since as I understand it `for` loops basically define a new scope at each iteration. The tests were failing before this change for me.